src/test/install_test.ts

```diff
@@ -54,6 +54,9 @@ suite('install', () => {
         async () => {
           let rejected = false;
           const someModulePath = path.join(os.tmpdir(), 'foo.js');
+          if (existsSync(someModulePath)) {
+            await fs.unlink(someModulePath);
```

Comment:
This test expects `someModulePath` to not exist at first. So I'm making sure it doesn't before running the test. On windows `os.tmpdir()` returns something like `C:\Users\andre\AppData\Local\Temp` so the directory is static across test runs.